classes/request.class.php

```diff
@@ -244,7 +240,7 @@ public function send_request($endpoint, $requestbody, $method, $requesttype = 'g
      */
     public function test_connection($apiurl, $apikey) {
 
-        $validurlregex = '/.+\.(turnitin\.com|turnitinuk\.com|turnitin\.dev|turnitin\.org|tii-sandbox\.com)\/api$/m';
+        $validurlregex = '/.+\.(turnitin\.com|turnitinuk\.com|turnitin\.dev|turnitin\.org|tii-sandbox\.com)$/m';
```

Comment:
Will this cause test_connection to fail if they put /api on the end? My thought was that the guides everywhere say to put /api on the end, and the support team tell people to put /api on the end, so the best solution would be to allow it both ways.